src/app/pages/cars/car/car.component.html

```diff
@@ -52,6 +52,24 @@
                 </mat-error>
               </mat-form-field>
             </div>
+            <div class="form-group" *ngIf="isBasic">
+              <div style="text-align: left">
+              <mat-checkbox [formControl]="isDefault">
```

Comment:
I think you have to add a default value here.
When you create a car and do not touch the checkbox the request looks like this:
_id: ""
vin: "W0L000051T2123456"
licensePlate: "PrivateNoDefault"
carCatalogID: "1271"
carCatalog: "Audi e-tron S"
isDefault: ""
isPrivate: "true"_

And the created entry like this:
![image](https://user-images.githubusercontent.com/33114317/81947520-f0107c00-9600-11ea-8226-6e8354fc4583.png)

Reply:
Like I mentioned in the backend PR. 
The request will not work anymore if you want to add a pool car, because you can just say if it is a private car or not and not f it is a company car or a pool car. 
Maybe you should take the type of the car in the request and not whether it is private or not.

Reply:
> Like I mentioned in the backend PR.
> The request will not work anymore if you want to add a pool car, because you can just say if it is a private car or not and not f it is a company car or a pool car.
> Maybe you should take the type of the car in the request and not whether it is private or not.

Yes true !
done